src/components/Review/Review/Review.tsx

```diff
@@ -0,0 +1,153 @@
+'use client'
+
+import React from 'react'
+import { useMediaQuery } from 'usehooks-ts'
+import { BiDislike, BiLike } from 'react-icons/bi'
+import { FaStar } from 'react-icons/fa'
+import { Review as ReviewType } from '@/types/ReviewType'
+import { formatReviewDate } from '@/components/Review/ReviewsList/formatReviewDate'
+import Button from '@/components/UI/Buttons/Button/Button'
+
+interface IReview {
+  isUserReview: boolean
+  review: ReviewType | null
+  toggleReviewExpansion?: (id: string) => void
+  deleteReview?: (id: string) => void
+  likeReview?: (id: string) => void
+  disLikeReview?: (id: string) => void
+  isExpanded?: boolean
+}
+
+const Review: React.FC<Readonly<IReview>> = ({
+  isUserReview = false,
+  review,
+  toggleReviewExpansion = () => {},
+  deleteReview = () => {},
+  likeReview = () => {},
+  disLikeReview = () => {},
+  isExpanded,
+}) => {
+  const isMediaQuery = useMediaQuery('(min-width: 768px)', {
```

Comment:
I would say it's better to use tailwind breakpoints for that, otherwise I would create some consts file with such breakpoints